client/common/cmdline.c

```diff
@@ -1489,13 +1480,31 @@ int freerdp_client_settings_parse_command_line_arguments(rdpSettings* settings,
 	}
 	else
 	{
-		CommandLineClearArgumentsA(args);
-
 		if (allowUnknown)
 		{
 			flags |= COMMAND_LINE_IGN_UNKNOWN_KEYWORD;
 		}
 
+		if (argc > 1)
+		{
+			const char* ext = strstr(argv[1], ".rdp");
+			const char* assist = strstr(argv[1], ".msrcIncident");
+
+			if (ext)
+			{
```

Comment:
Perhaps we should check that it is ending by .rdp, than just checking that the filename contains .str